var/spack/repos/builtin/packages/amduprof/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Amduprof(Package):
+    """AMD uProf is a software profiling analysis tool for x86 applications
+    and provides event information unique to the AMD "Zen"-based processors."""
+
+    homepage = "https://developer.amd.com/amd-uprof/"
+    url      = "https://developer.amd.com/wordpress/media/files/AMDuProf_Linux_x64_3.5.671.tar.bz2"
+
+    version('3.5.671', sha256='a73066305228658a14af5ecd6cf45a1aa47ae94f7e9d14db31f43013d3ef1a43')
+
+    depends_on('binutils', type='run')
```

Comment:
Can you explain the dependency on `binutils`? Asking because of https://spack.readthedocs.io/en/latest/getting_started.html#binutils

Reply:
I ran the profiler on a clean node and it threw warnings about not being able to run `strings`.

Reply:
Can you please remove the dependency then? I think it enters in the description above somehow. If a software relies on `string` to be available and it isn't we shouldn't be imposing a dependency on `binutils`.

Reply:
`strings` is in `binutils`:
```bash
$ rpm -qf `which strings`
binutils-2.30-108.el8_5.1.x86_64
$ spack load binutils%gcc@8.5.0
$ which strings
~/spack/opt/spack/linux-rocky8-zen/gcc-8.5.0/binutils-2.38-75rlqjfboe4ah4nuai64lhjva2ak46ar/bin/strings
```

Reply:
It's also a `run` dependency, `binutils` should only break stuff when the linker loads the wrong `libbfd.so`.